keras/engine/training.py

```diff
@@ -928,6 +929,21 @@ def run_eagerly(self):
     def run_eagerly(self, value):
         self._run_eagerly = value
 
+    @property
+    def distribute_reduction_method(self):
+        """Settable attribute indicating how the model should reduce
+        loss and metric values from replicas.
+
+        Default: `"auto"`. This should be good for all cases.
+        It boils down to using `"sum"` or `"first"` conditioned on
```

Comment:
instead of "conditioned on whether TPU is used", I'd say "conditioned on the specific implementation of the `tf.distribute` strategy".

Also, if possible, can you describe in the docstring in what cases (such as the case you have here), using `sum` will make sense?

Reply:
changed the wording and added a sentence about the situation for `sum` - https://github.com/keras-team/keras/pull/16664/commits/db2aba0168d5481e6c78cb90f5652a116b0ada40